ansible/configs/ansible-tower-implementation/default_vars.yml

```diff
@@ -0,0 +1,78 @@
+---
+
+###### VARIABLES YOU SHOULD CONFIGURE FOR YOUR DEPLOYEMNT
+###### OR PASS as "-e" args to ansible-playbook command
+
+### Common Host settings
+env_type: satellite-vm 
+output_dir: /tmp/workdir                # Writable working scratch directory
+email: "{{env_type}}@example.com"
+
+## guid is the deployment unique identifier, it will be appended to all tags,
+## files and anything that identifies this environment from another "just like it"
+guid: defaultguid
+
+### Other Options are: file, satellite and rhn
+repo_method: file 
+use_own_repos: true
+repo_version: "6.4"
+### For RHN login 
+# repo_method: rhn
+# rhsm_pool_ids:
+#   - 8a85f99b6b498682016b521dfe463949
+# rhel_subscription_user: 
+# rhel_subscription_pass: 
+### If using repo_method: satellite, you must set these values as well.
+# satellite_url: satellite.example.com
+# satellite_org: Sat_org_name
+# satellite_activationkey: "rhel7basic"
+######
+repo_version: "3.6.0-1"
```

Comment:
You have `repo_version` var defined twice. See line 18 also.

Reply:
Fixed